.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: 'release'
+on: [workflow_dispatch]
```

Comment:
is there more custom config needed here?

Reply:
the only thing we need for every repo is a workflow to execute this, but it can be an exact copy of this one